xs/src/libslic3r/ConfigBase.cpp

```diff
@@ -931,7 +936,7 @@ ConfigOptionPoint3::deserialize(std::string str, bool append) {
         this->value.y = boost::lexical_cast<coordf_t>(tokens[1]);
         this->value.z = boost::lexical_cast<coordf_t>(tokens[2]);
     } catch (boost::bad_lexical_cast &e){
-        std::cout << "Exception caught : " << e.what() << std::endl;
+        std::cout << "Config option deserialisation error of ["<<str<<"]  : " << e.what()<<" (expected 3D point)" << std::endl;
```

Comment:
Prefer to use Slic3r::Log, although I would prefer to push more information down into `what()` if at all possible.